src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -2325,22 +2327,32 @@ public void TransformStrStr12()
                     iCount++;
                 }
 
-                try
-                {
-                    xslt.Transform(szFullFilename, "\\\\");
-                }
-                catch (System.Exception)
-                {
-                    iCount++;
-                }
             }
 
-            if (iCount.Equals(3))
+            if (iCount.Equals(2))
                 return;
             _output.WriteLine("Exception not generated for invalid ouput destinations");
             Assert.True(false);
         }
 
+        //[Variation("Output filename is \'\\\\\'")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [InlineData()]
+        [Theory]
+        public void TransformStrStr12_win()
+        {
+            String szFullFilename = FullFilePath("fruits.xml");
+
+            if (LoadXSL("showParam.xsl") == 1)
+            {
+                Assert.Throws<System.ArgumentException>(() => xslt.Transform(szFullFilename, "\\\\"));
+                return;
+            }
+
+            _output.WriteLine("Exception not generated for invalid ouput destination");
+            Assert.True(false);
```

Comment:
(Looks like there are a bunch of other places that would apply, too, so maybe a subsequently cleanup)